ts/client/src/accounts/oracle.ts

```diff
@@ -133,3 +155,26 @@ export function isSwitchboardOracle(accountInfo: AccountInfo<Buffer>): boolean {
 export function isPythOracle(accountInfo: AccountInfo<Buffer>): boolean {
   return accountInfo.data.readUInt32LE(0) === PythMagic;
 }
+
+export function checkOracleConfidenceAndStaleness(
```

Comment:
It's surprising to me that this is true when it's stale or inconfident. Could this be `isOracleStaleOrUncofident()`? Or reverse the output and use `isOracleHealthy` or something

Reply:
changed to isOracleStaleOrUncofident